avocado/plugins/runner_nrunner.py

```diff
@@ -33,6 +33,7 @@
 from avocado.core.task.runtime import RuntimeTask
 from avocado.core.task.statemachine import TaskStateMachine, Worker
 from avocado.core.test_id import TestID
+from avocado.core.teststatus import mapping
```

Comment:
Just a slightly off question here: Maybe later on this could be turned into `test_status` or in reverse `test_id` could be turned into `testid`. I am also curious what is the current separation between the `avocado.core.result` and `avocado.core.teststatus` modules? Don't we have some redundancy here?

Reply:
@pevogam my apologies, I only saw your comments after I merged this PR.

Reply:
> Just a slightly off question here: Maybe later on this could be turned into `test_status` or in reverse `test_id` could be turned into `testid`. I am also curious what is the current separation between the `avocado.core.result` and `avocado.core.teststatus` modules? Don't we have some redundancy here?

Yes, all of those points you raised seem to be good opportunities for improvements. There's a LOT to be improved here.  We're carrying a lot of baggage, but after LTS, we'll drop as much as needed.